dbschema/budget-record.esdl

```diff
@@ -0,0 +1,13 @@
+module default {
+  type BudgetRecord extending Project::ContextAware {
```

Comment:
Technically these are project children too, so they should be represented as such.
```suggestion
  type BudgetRecord extending Project::Child {
```